pkg/templates/charts/toggle/console-mce/templates/console-quickstart.yaml

```diff
@@ -0,0 +1,136 @@
+      review:
+        failedTaskHelp: >-
+          This task is incomplete. Try the task again.
+        instructions: |-
+          Verify that the console plug-in was enabled successfully:
+
+            - In the navigation menu, is the first item a menu containing **All clusters** and **local-cluster**?
+      summary:
+        failed: >-
+          This task is incomplete. Try the task again, or read more about this
+          topic.
+        success: >-
+          You just enabled the console plug-ins! Next, we'll enable the feature
+          gate.
+      title: Enable console plug-in
+    - description: >
+
+        **Warning**: This enables Technology Preview features 
+          that are not part of the default features. Enabling this feature set 
+          cannot be undone and prevents upgrades. **This feature set is NOT 
+          recommended on production clusters**.
+
+        To enable feature gate:
+
+        1. In the OpenShift Container Platform web console, switch to **Administration > Cluster settings**
+
+        1. On the **Configuration** tab, click **FeatureGate**. 
+
+        1. On the **YAML** tab, edit the cluster instance to add the following
+        specific feature sets:
+
+
+            ```
+            spec:
+              featureSet: TechPreviewNoUpgrade
+            ```
+
+        1. Click **Save**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: |
+          Verify that the feature gate was enabled successfully:
+
+            - In the navigation menu, click **Administration > Cluster settings > 
+            Configuration > FeatureGate > YAML**.
+
+            - Do you see **TechPreviewNoUpgrade**?  
+      summary:
+        failed: >-
+          Check your work to make sure that the Knative Service resource was
+          created.
+        success: You just created an instance of the Knative Service resource.
+      title: Enable feature gate
+    - description: >
+
+        1. From the Administrator perspective, go to **OperatorHub** from
+        the **Operators** section of the navigation.
+
+        1. In the **Filter by keyword** field, type “identity configuration
+        management for Kubernetes”.
+
+        1. Click the **identity configuration management for Kubernetes** tile to open the Operator details.
+
+        1. At the top of the **identity configuration management for Kubernetes**
+        panel that opens, click **Install**.
+
+        1. Fill out the Operator subscription form by selecting the channel and 
+        namespace that match your OpenShift cluster, and then click Install.
+
+
+        1. On the **Installed Operators** page, wait for the status of **identity
+        configuration management for Kubernetes** to change from
+        **Installing** to **Succeeded**.
+      review:
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+        instructions: >
```

Comment:
Fill out the _Operator subscription_ form by selecting the channel and namespace that match your OpenShift cluster. Then click **Install**.

On the _Installed Operators_ page, wait for the status of the _identity configuration management for Kubernetes_ to update from _Installing_ to _Succeeded_

Reply:
Adopted this, but have since updated the content somewhat.